docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/import-eks-cluster-enable-hybrid-mode.md

```diff
@@ -0,0 +1,517 @@
+
+20. Replace `<vpcCidrIp>` with your hybrid node CIDR list. For example, `10.200.0.0/16`. The `vpcCIDR` defines the
+    network range for VPC resources to ensure proper routing through the VPN.
+
+21. Apply the ConfigMap to your cluster.
+
+    Example command.
+
+    ```shell hideClipboard
+    kubectl create -f=/dev/stdin <<-EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: hybrid-config
+      namespace: kube-system
+    data:
+      serviceCIDR: "192.168.0.0/16"
+      vpcCIDR: "10.200.0.0/16"
+    EOF
+    ```
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. Navigate to the left **Main Menu** and select **Clusters**.
+
+3. Click on the cluster you created to view its details in the **Overview** tab.
+
+4. Ensure the **Cluster Status** field displays **Running**.
+
+## Configure CNI for Hybrid Nodes
+
+You need to configure a CNI for Amazon EKS Hybrid Nodes as the standard Amazon EKS VPC CNI lacks hybrid node support.
+Cilium handles IP Address Management (IPAM) and Border Gateway Protocol (BGP) for non-AWS nodes.
+
+### Prerequisites
+
+- Your Palette account role must have the `clusterProfile.create` permission to import a cluster profile. Refer to the
+  [Cluster Profile](../../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile)
+  permissions for guidance.
+
+If enabling [Cilium Envoy](https://docs.cilium.io/en/latest/security/network/proxy/envoy/) or other Cilium add-ons, you
+must also complete the following prerequisites:
+
+- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/) is installed and available in your local workstation.
+
+- Access to your Amazon EKS cluster through kubectl.
+
+  - To access your cluster with kubectl, you can use the AWS CLI's built-in authentication capabilities. If you are
+    using a custom OIDC provider, you will need to configure your kubeconfig to use your OIDC provider.
+
+    Refer to the [Access Imported Cluster with Kubectl](#access-imported-cluster-with-kubectl) section for more
+    information.
+
+### Add CNI Cluster Profile
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Profiles**.
+
+3. On the **Profiles** page, click **Add Cluster Profile**.
+
+4. Fill out the basic information and ensure **Type** is set to **Add-on**. Click **Next** when done.
+
+5. In **Profile Layers**, click **Add New Pack**.
+
+6. Enter **Cilium** in the search box, and select it. It appears in the **System App** category and must be version
+   **1.16.0** or above.
+
+7. Click the **Presets drop-down Menu**.
+
+8. For **IPAM mode**, select **Cluster Pool**.
+
+9. In the YAML editor, search for **clusterPoolIPv4PodCIDRList**. This parameter specifies the overall IP ranges
+   available for pod networking across all your hybrid nodes.
+
+Adjust the pod CIDR list for hybrid pods in other networks that need to connect to this cluster. For example,
```

Comment:
Nest back into list

Reply:
Not sure how this happened, but thanks for catching!